### PR TITLE
Bug fixes in existing image restriction functions, additional phone models added to yaml file, and first draft horizon detector/remover. 

### PR DESCRIPTION
**Pull request by wingarlo**
I am fairly certain I will have to clean my code a bit so it will pass the checks.

As mentioned in the update email from Jan 24th:

> -is_size states that it only allows up to 400kb, but the yaml file requires <= 4Mb
> -is_res states that is allows between 1x1 - 1000x1000, but the yaml file requires between 600x600 - 6000x6000
> -is_type doesn't allow JPG because of case sensitivity even though it is the same as jpg
> -is_edited gets passed arguments in the wrong order (date modified gets assigned to date taken and vice versa). This has no effect on how it runs at all, but I fixed it anyway because it bugs me.

These bugs/errors in the existing image restriction functions have been fixed. 

I have tested the image restriction functions with a few more phone models and if they successfully determined the validity of the image, I added the model name to the yaml config file. 

Lastly, as you'll see in issue #107 I have written an algorithm (this algorithm is pretty inaccurate right now) that checks for intensity drops in the horizontal rows and if there is a great enough one, it crops the image at that row in order to isolate the sky.

### CODE REVIEW

**Comment by wingarlo** [comment]
A couple side notes:

- Not sure why the commits to my branch go so far back, and if this is an error, I'll try to fix it.
- In retrospect, this should have been over the course of a couple/few pull requests.

**Comment by kwhitehall** [comment]
Thanks for the PR @wingarlo. Reviewing ...

**Comment by wingarlo** [comment]
The split-image-bgr-data is just data from my algorithm that I use to adjust the algorithm to make it more accurate. I'll remove that as it shouldn't be in the repo.

The exiftest was just a test to get used to the exif retrieval functions and wasn't supposed to be pushed, so I will remove that.

I will lint the files, remove the print statements as they are just for testing purposes, and update the pull request.

File: aerolyzer/horizon.py [hunk 1]
```diff
@@ -3,43 +3,63 @@
...
-	    x2 = int(x0 - 1000*(-b))
-	    y2 = int(y0 - 1000*(a))
-	    cv2.line(img_noblur,(x1,y1),(x2,y2),(0,255,0),2)
-	cv2.imwrite("results/H_"+image,img_noblur)
-
-#horizon()._testAnalyze("./images/")
-			
+
+def is_sky(a, path):
+    #print path
```
Comment (kwhitehall) [inline aerolyzer/horizon.py]: remove these commented out bits of code through out the files in this PR

**Comment by kwhitehall** [comment]
Thanks @wingarlo. Going to merge. 
Will open some issues towards rebasing as previously discussed.

**Pull request merged**